docs/notifications.md

```diff
@@ -80,7 +80,7 @@ Recognized stream names:
  * `notary_request_event`
    Filter: `sender` field containing a string with hex-encoded Uint160 (LE
    representation) for notary request's `Sender` and/or `signer` in the same
-   format for one of main transaction's `Signers`.
+   format for one of main transaction's `Signers`. `Type` field containing a string with event type, which could be one of "added" or "removed".
```

Comment:
s/`Type`/`type`, pay attention to the case. Documentation tells about user-facing values, and user-facing name of this field starts with lowercase letter.

Reply:
fixed